sdk/core/System.ClientModel/src/ModelReaderWriter/ModelReaderWriterOptions.cs

```diff
@@ -1,14 +1,21 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
 namespace System.ClientModel.Primitives;
 
 /// <summary>
 /// Provides the client options for reading and writing models.
 /// </summary>
 public class ModelReaderWriterOptions
 {
+    // Proxies are compared by exact type only.
```

Comment:
Would it be possible to add another sentence to this comment describing the implications of this?  i.e. "this means that TryGetProxy will not find a proxy if it has been registered by a base type or derived type."  I think that would help maintainers understand the thrust of this comment better.